docs/source/WritingTests/WritingSimpleTests.rst

```diff
@@ -167,6 +162,48 @@ to pick up a living guest, connect to it via ssh, and return its uptime.
            logging.info("Guest uptime result is: %s", uptime)
            session.close()
 
+#. Note that all failures that might happen here are implicitly handled
+   by the methods called. If a test went from its beginning to its end
+   without unhandled exceptions, avocado assumes the test automatically
+   as **PASS**, *no need to mark a test as explicitly passed*. If you
+   have explicit points of failure, for more complex tests, you might
+   want to mark it explicitly. ``test.cancel`` makes the test **CANCEL**
+   (it is a new feature which not supported in avocado 36lts, you may
+   want to use ``test.skip`` to make the test **SKIP** to achieve the
+   similar purpose), ``test.error`` makes the test **Error**, and
+   ``test.fail`` makes the test **Fail**. And, in recent Avocado version
+   (since commit 7ecf09fa), there're three exceptions: ``TestFail``,
+   ``TestError``, and ``TestCancel`` are added to avocado namespace,
```

Comment:
there'`re` ... `are` added. => you have two `are`s in one sentence. How about

    ...since commit 7ecf09fa) three exceptions: ... were added to ...